lib/ProductOpener/Display.pm

```diff
@@ -7749,15 +7755,14 @@ HTML
 
 			my $ilc = $ingredients_text_lang;
 
-
 			$html .= <<HTML
 
-<div class="button_div" id="editingredientsbuttondiv"><button id="editingredients" class="small button" type="button">Edit ingredients ($ilc)</div>
+<div class="button_div" id="editingredientsbuttondiv"><button id="editingredients" class="small button" type="button">Edit ingredients$ingredients_text_lang_html</div>
```

Comment:
```suggestion
<div class="button_div" id="editingredientsbuttondiv"><button id="editingredients" class="small button" type="button">Edit ingredients $ingredients_text_lang_html</div>
```

Reply:
`$ingredients_text_lang_html` already contains a space

Reply:
Just noticed that these changes have broke Power User Script that needed to read `($ilc)` to call Google Cloud Vision. Of course "Spanish" is better for humans. But could you add a `data-lc="es"` in the `<div id="editingredients"`>?

Sorry to see that so late...